### DISCUSSION TITLE
GraphicRaycast를 직접 쓸수 있나요?

### DISCUSSION THREAD

**Discussion by pda017** [naverz/zepeto-studio-kor#285] [5. Q&A]
UGUI GraphicRaycast를 써서 직접 ui에 레이케스팅을 하고 싶은데 어떻게 써야 할지 모르겠네요.
애초에 쓸수 있는지 궁금하네요.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

import {GraphicRaycaster} from 'UnityEngine.UI' 

위와 같이 import하면 GraphicRaycaster를 사용할 수 있습니다.
